.github/workflows/dev-image-update.yaml

```diff
@@ -69,21 +69,23 @@ jobs:
           DISABLE_ESLINT_PLUGIN: true
         run: |
           rm -rf jade-dev-account.pem
-          docker build -t gcr.io/${{ env.gcr_google_project }}/jade-data-repo-ui:${GCR_TAG} --build-arg CACHEBUST=$(date +%s) .
+          docker build -t gcr.io/${gcr_google_project}/jade-data-repo-ui:${GCR_TAG} --build-arg CACHEBUST=$(date +%s) .
```

Comment:
As I understand it, the reason for this fix is to replace github workflows template expansion with bash shell expansion, since bash shell supports quoting to prevent code injection. But this new expression doesn't use bash quoting, so I believe it would also be vulnerable.
```suggestion
          docker build -t gcr.io/"${gcr_google_project}"/jade-data-repo-ui:"${GCR_TAG}" --build-arg CACHEBUST="$(date +%s)" .
```

It's probably good to apply all recommended fixes, although in these cases I think you could use logic to determine that the expansions only refer to objects which we control and therefore wouldn't be a possible injection attack vector.